modes/vi-mode/core.lisp

```diff
@@ -84,23 +111,21 @@
         (if (symbolp state)
             (get state 'state)
             state))
-  (assert (vi-state-p state))
+  (assert (or (subtypep (type-of state) 'vi-state) (typep 'vi-state (type-of state))))
```

Comment:
Does this mean `(typep state 'vi-state)` ?

Reply:
Yes, nice catch!